docs/00 - Stellar Anchor Platform.md

```diff
@@ -0,0 +1,98 @@
+logic necessarry to enable interoperability with the Stellar network.
+
+This way, Anchors willing to integrate with Stellar can use the Anchor Platform
+to speed up their development process, allowing them to focus mainly on the
+business logic that's specific to their businesses and use cases.
+
+## Table of Content
+
+The full documentation can be found under the [`docs` directory](/docs), under the structure:
+
+- [00 - Stellar Anchor Platform](/docs/00%20-%20Stellar%20Anchor%20Platform.md)
+- [01 - Running & Configuring the Application](/docs/01%20-%20Running%20%26%20Configuring%20the%20Application)
+  - [A - Running & Configuring the Application](/docs/01%20-%20Running%20%26%20Configuring%20the%20Application/A%20-%20Running%20%26%20Configuring%20the%20Application.md)
+  - [B - Circle Payment Observer](/docs/01%20-%20Running%20%26%20Configuring%20the%20Application/B%20-%20Circle%20Payment%20Observer.md)
+- [02 - Contributing](/docs/02%20-%20Contributing)
+  - [A - CONTRIBUTING.md](/docs/02%20-%20Contributing/A%20-%20CONTRIBUTING.md)
+  - [B - Developer Tools](/docs/02%20-%20Contributing/B%20-%20Developer%20Tools.md)
+  - [C - Logging Guidelines](/docs/02%20-%20Contributing/C%20-%20Logging%20Guidelines.md)
+  - [D - Database Migration](/docs/02%20-%20Contributing/D%20-%20Database%20Migration.md)
+  - [E - Publishing the SDK](/docs/02%20-%20Contributing/E%20-%20Publishing%20the%20SDK.md)
+- [03 - Anchor Integration](/docs/03%20-%20Anchor%20Integration) // TODO
+- [04 - Subprojects Usage](/docs/04%20-%20Subprojects%20Usage) // TODO
+
+## Glossary
+
+Here are the important terminology used in this project:
+
+- **Anchor**: on/off ramps of the Stellar network. More information is available [here](https://developers.stellar.org/docs/anchoring-assets/).
+- **Wallet**: a frontend application used to interact with the Stellar network on behalf of a user.
+- **Sending Anchor**: a therminology used in the context of [SEP-31]. Refers to an entity that receives funds from a user and forwards it (after taking a fee) to a receiving anchor, in the SEP-31 `Sending Client->Sending Anchor->Receiving Anchor-> Receiving Client` flow.
+- **Receiving Anchor**: a therminology used in the context of [SEP-31]. Refers to an entity that receives funds from a user and forwards it (after taking a fee) to a receiving anchor, in the SEP-31 `Sending Client->Sending Anchor->Receiving Anchor-> Receiving Client` flow. It's the entity the Platform SEP-31 service is currently built for.
+- **Ecosystem Players**: any entity or end-user that's part of the Stellar ecosystem. This includes Anchors, Wallets, Users, Issuers, Payment providers, and others.
+- **Anchor Platform (or Platform)**: the web application that will be exposing public endpoints and APIs. It will be compliant with the [SEPs] to guarantee interoperability in the Stellar network and will delegate business-specific logic to the Anchor Server.
```

Comment:
grammar nit: can we change the phrasing "will be" (future-tense) to "is" (present-tense)? This happens a few times in the doc.

```suggestion
- **Anchor Platform (or Platform)**: the web application that will be exposing public endpoints and APIs. It is compliant with the [SEPs] to guarantee interoperability in the Stellar network and delegates business-specific logic to the Anchor Server.
```